buildvertical.targets

```diff
@@ -81,6 +107,20 @@
      </PropertyGroup>
   </Target>
 
+  <!-- Runs in a leaf project (eg: csproj) to determine configurations to package -->
+  <Target Name="GetPackageConfigurations"
+          Returns="$(PackageConfigurations)">
```

Comment:
This enables an optionally independent property for the configurations that go in the package.  I'm open to other knobs here.  Perhaps a better semantic would be ExcludePackageConfigurations, where PackageConfigurations = BuildConfigurations - ExcludePackageConfigurations.